src/message_subscriber.cpp

```diff
@@ -131,10 +136,10 @@ code message_subscriber::load(message_type type, uint32_t version,
         CASE_RELAY_MESSAGE(stream, version, get_block_transactions);
         CASE_RELAY_MESSAGE(stream, version, get_data);
         CASE_RELAY_MESSAGE(stream, version, get_headers);
-        CASE_RELAY_MESSAGE(stream, version, headers);
+        CASE_RELAY_MESSAGE_SETTINGS(stream, version, headers, settings);
```

Comment:
You can’t use relay here, the messages must still be handled.

Reply:
I guess this comment belongs to line 127. Anyway, good catch! Copy&paste is the biggest source of error...

Reply:
Yes

Reply:
Fixed.